smartcosmos-framework/src/main/java/net/smartcosmos/annotation/SmartCosmosBootstrapConfiguration.java

```diff
@@ -80,16 +79,13 @@ public SendsSmartCosmosEventAdvice sendsSmartCosmosEventAdvice(SmartCosmosEventT
         @Bean
         @Autowired
         @Profile("!test")
-        SmartCosmosEventTemplate smartCosmosEventTemplate(
-            RibbonClientHttpRequestFactory ribbonClientHttpRequestFactory,
-            Executor smartCosmosEventTaskExecutor) {
+        SmartCosmosEventTemplate smartCosmosEventTemplate(Executor smartCosmosEventTaskExecutor) {
```

Comment:
However, this will definitely break the API, since all of the dependencies on Eureka are dropped from the Framework.